src/ralph/virtual/subscriptions.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+import pkg_resources
+import pyhermes
+from django.conf import settings
+from django.utils.text import slugify
+
+
+
+logger = logging.getLogger(__name__)
+
+
+ENTRY_POINTS_GROUP = 'ralph.dchost_processors'
```

Comment:
so is it dchost processor or cloud processor?

Reply:
It's a cloud processor, I have to rename it.